bip-schnorr.mediawiki

```diff
@@ -110,6 +110,11 @@ The following convention is used, with constants as defined for secp256k1:
 *** Return the unique point ''P'' such that ''x(P) = x'' and ''y(P) = y''.
 ** The function ''point(x)'', where ''x'' is a 32-byte array, returns the point ''P = lift_x(int(x))''.
 ** The function ''hash(x)'', where ''x'' is a byte array, returns the 32-byte SHA256 hash of ''x''.
```

Comment:
Now with both `hash` and `hash_tag`, I thin renaming `hash` to `sha256` is a good idea. It's just clearer.
(Maybe I said this already but I can't remember. Feel free to ignore if there were good arguments against it and we discussed this already...)

Reply:
removed `hash`